packages/cosmos-bin/src/main.rs

```diff
@@ -388,10 +388,28 @@ impl Subcommand {
                 dest,
                 coins,
             } => {
+                // gas is wildly underestimated on osmosis testnet at least
+                // bump up the multiplier to make sure we have enough
+                // then set it back when we're done sending coins
+                let original_gas_multiplier = cosmos.get_gas_multiplier();
+                if original_gas_multiplier < 1.5 {
+                    log::info!(
```

Comment:
This would probably get really noisy really fast. How about debug level?

Also: what about making the gas multiplier a CLI flag with env var override?